content/cloud/noops-sql-db.md

```diff
@@ -34,24 +35,21 @@ Once you have access to the private beta, please ensure you have [Spin CLI](./cl
 * Your NoOps SQL Database can hold up 1 GB of data
 
 *Service Limitations*
-* The `--sqlite` flag not supported on `spin cloud deploy`
+* The `--sqlite` flag is not supported on `spin cloud deploy`
 * Fermyon **Cloud** only supports databases named "default" at this time
 
-## How To:
+## How To
 
-### Prepare a Database
+Firstly, if you have not already done so, you will need to [create your first Spin application](https://developer.fermyon.com/spin/quickstart#create-your-first-application). Our blog posts [Announcing Fermyon Cloud’s NoOps SQL Database](https://www.fermyon.com/blog/announcing-noops-sql-db) and [NoOps and Serverless Are the Perfect Pair](https://www.fermyon.com/blog/noops-and-serverless-are-the-perfect-pair) each have an example of how to create a NoOps SQLite Storage application. To tell Spin that we want to use SQLite storage, we only need to [grant SQLite permission to a component in the application’s manifest]((https://developer.fermyon.com/spin/sqlite-api-guide#granting-sqlite-database-permissions-to-components)) (the spin.toml file). For example:
 
-You can use the `spin cloud sqlite execute` command to check if a table needs to be created, and if so create one: 
-
-```bash
-$ spin cloud sqlite execute inspirational-pig "CREATE TABLE IF NOT EXISTS todos (id INTEGER PRIMARY KEY AUTOINCREMENT,description TEXT NOT NULL,due_date DATE,starred BOOLEAN DEFAULT 0,is_completed BOOLEAN DEFAULT 0)"
+```
+[component]
+sqlite_databases = ["default"]
 ```
 
->> If you need to look up the name of your database, use the `spin cloud sqlite list` command
-
-### Manage the Database Lifecycle
+### Database Access From Many Components
 
-A database is created when you deploy a Spin application that references a "default" SQLite database in its component manifest. Multiple components in the same Spin application can be given access to the default database like so:
+A database is created when you build a Spin application referencing a "default" SQLite database in the component section of an application's manifest. Multiple components in the same Spin application can be given access to the same `default` database like so:
```

Comment:
The more I look at this, the more I think we should drop the stuff about "multiple components" - it just recapitulates one of the less interesting parts of the Spin docs, and points readers in the wrong direction.  We could show _one_ component if we really wanted to but even then I kind of feel we should decouple thing from the "here is how the database gets created" stuff.

Reply:
Agreed, this has been dropped.